### PR TITLE
Update image_processing.facebox.markdown

### PR DESCRIPTION
**Pull request by robmarkcole**
**Description:**
Adds info on events, automations and limiting processing

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Comment by robmarkcole** [comment]
OK theres something wrong with this automation - fires for EVERY face, not just Ringo. But the use of 
```yaml
  trigger:
  - event_data:
      name: 'Ringo_Starr'
```
Means it should only fire for Ringo? @dgomes can you verify?

**Review by fabaff — DISMISSED** [review]
Thanks :bird:

**Comment by robmarkcole** [comment]
Ok the automation is correct, the issue is in the actual component. Corrected in https://github.com/robmarkcole/HASS-Machinebox-Facebox/blob/adds-events/custom_components/image_processing/facebox.py

**Comment by robmarkcole** [comment]
@MartinHjelmare how's this? Automation is fine after latest PR

File: source/_components/image_processing.facebox.markdown [hunk 1]
```diff
@@ -13,14 +13,21 @@ featured: false
 ha_release: 0.70
 ---
 
-The `facebox` image processing platform allows you to detect and recognise faces in a camera image using [Facebox](https://machinebox.io/docs/facebox). The state of the entity is the number of faces detected, and recognised faces are listed in the `matched_faces` attribute. Facebox runs in a Docker container, and it is recommended that you run this container on a machine with a minimum of 2 GB RAM. On your machine with Docker, run the Facebox container with:
-```
+The `facebox` image processing platform allows you to detect and recognise faces in a camera image using [Facebox](https://machinebox.io/docs/facebox). The state of the entity is the number of faces detected, and recognised faces are listed in the `matched_faces` attribute. An `image_processing.detect_face` event is fired for each recognised face, and the event `data` provides the `confidence` of recognition, the `name` of the person, the `id` of the image associated with the match, the rectangle `rect` that contains the face in the image, and the `entity_id` that processing was performed on.
```
Comment (arsaboo) [inline source/_components/image_processing.facebox.markdown]: `recognise` -> `recognize`

File: source/_components/image_processing.facebox.markdown [hunk 2]
```diff
@@ -58,3 +65,29 @@ source:
...
+{% raw %}
+```yaml
+- id: '1120092824666'
+  alias: Ringo Starr recognised
+  trigger:
+  - event_data:
+      name: 'Ringo_Starr'
+    event_type: image_processing.detect_face
+    platform: event
+  condition: []
```
Comment (arsaboo) [inline source/_components/image_processing.facebox.markdown]: Remove empty `condition`

File: source/_components/image_processing.facebox.markdown [hunk 3]
```diff
@@ -58,3 +65,29 @@ source:
       required: false
       type: string
 {% endconfiguration %}
+
+## {% linkable_title Automations %}
+
+Use the `image_processing.detect_face` events to trigger automations, and breakout the `trigger.event.data` using a [data_template](https://www.home-assistant.io/docs/automation/templating/). The following example automation sends a notification when Ringo Star is recognised:
+
+{% raw %}
+```yaml
```
Comment (arsaboo) [inline source/_components/image_processing.facebox.markdown]: Please adjust the sequence in the YAML. I know it came from the automation editor, but that is not optimal.
```yaml
  trigger:
    - platform: event
      event_type: image_processing.detect_face
      event_data:
        name: 'Ringo_Starr'
```

File: source/_components/image_processing.facebox.markdown [hunk 4]
```diff
@@ -58,3 +65,29 @@ source:
...
+  - event_data:
+      name: 'Ringo_Starr'
+    event_type: image_processing.detect_face
+    platform: event
+  condition: []
+  action:
+  - data_template:
+      message: Ringo_Starr recognised with probability {{ trigger.event.data.confidence }}
+      title: Door-cam notification
+    service: notify.platform
```
Comment (arsaboo) [inline source/_components/image_processing.facebox.markdown]: Same, adjust the sequence. `service` comes before `data_template`

File: source/_components/image_processing.facebox.markdown [hunk 5]
```diff
@@ -58,3 +65,29 @@ source:
...
+```
+{% endraw %}
+
+## {% linkable_title Optimising resources %}
+
+[Image-classifier components](https://www.home-assistant.io/components/image_processing/) process the image from a camera at a fixed period given by the `scan_interval`. This leads to excessive processing if the image on the camera hasn't changed, as the default `scan_interval` is 10 seconds. You can override this by adding to your config `scan_interval: 10000` (setting the interval to 10,000 seconds), and then call the `image_processing.scan` service when you actually want to perform processing.
```
Comment (arsaboo) [inline source/_components/image_processing.facebox.markdown]: `Image-classifier components` -> `Image processing components`

**Review by arsaboo — CHANGES_REQUESTED** [review]

File: source/_components/image_processing.facebox.markdown (hunk 5 above)
Comment (robmarkcole) [reply source/_components/image_processing.facebox.markdown]: On second thoughts, perhaps this should be placed in https://www.home-assistant.io/components/image_processing/ as it applies to all `image_processing` components..?

File: source/_components/image_processing.facebox.markdown (hunk 3 above)
Comment (robmarkcole) [reply source/_components/image_processing.facebox.markdown]: ok

File: source/_components/image_processing.facebox.markdown (hunk 4 above)
Comment (robmarkcole) [reply source/_components/image_processing.facebox.markdown]: ok

File: source/_components/image_processing.facebox.markdown (hunk 2 above)
Comment (robmarkcole) [reply source/_components/image_processing.facebox.markdown]: ok

File: source/_components/image_processing.facebox.markdown (hunk 5 above)
Comment (robmarkcole) [reply source/_components/image_processing.facebox.markdown]: ok

File: source/_components/image_processing.facebox.markdown (hunk 1 above)
Comment (robmarkcole) [reply source/_components/image_processing.facebox.markdown]: ok..

**Comment by robmarkcole** [comment]
Changes made @arsaboo

**Review by arsaboo — APPROVED** [review]

**Comment by arsaboo** [comment]
I am guessing this is waiting for https://github.com/home-assistant/home-assistant/issues/14950 to be merged.

**Comment by robmarkcole** [comment]
@arsaboo it can merge now, the automation works in `current` and `next`

**Pull request merged**